app/(admin)/admin/notifications/page.tsx

```diff
@@ -42,30 +47,87 @@ const AdminNotifications = (): JSX.Element => {
     await sendEmailNotifications({ content, groupUsers, subject });
   };
 
+  /**
+   * Function to handle radio selection logic.
+   * @param value - Value of the radio buttons.
+   */
+  const handleRadioChange = (value: string): void => {
+    if (value === 'all') {
```

Comment:
Correct me if I'm wrong, but is adding this conditional outside of this tickets scope?

Reply:
As mentioned in discord, this was an addition by @alexappleget as this is branched off of his branch and we wanted to expedite this process.